lib/services/groups/groupConfigService.js

```diff
@@ -0,0 +1,320 @@
+/*
```

Comment:
Is this file a copy of lib/services/groups/groupService.js? 
IMHO In order to keep it as simple as possible a solution with just one implementation will be desirable.

Reply:
I agree with @AlvaroVega 

I understand the need of a "parallel" file comes from the requeriment of having a different top-level payload field different for each case. I mean, `POST /iot/services uses` "services" as top-level element and `POST /iot/configGroup` uses "configGroup".

However, there should be smarter ways of dealing with this than duplicate a whole file...

Reply:
Hi @AlvaroVega, @fgalan,
I have created the parallel file because if at the end /iot/services is removed then we just need to delete lib/services/groups/groupService.js file. In case of single file implementation, we need to delete the specific implementation of /iot/services.

Please let me know, if I should create a single implementation for both the endpoints (/iot/services and /iot/configGroups)?
Thank you.

Reply:
A single implementation for both endpoints is desirable.

Reply:
> A single implementation for both endpoints is desirable.

I agree.

I understand your point @Madhu-NEC about "easy removal of old stuff" when we will be ready for that. But it should be also possible even in the case of using a single implementation. Let's see how that single implementation is at the end to provide more specific feedback on this.

Reply:
Hi @AlvaroVega , @fgalan ,
Fixed in cffa723

Reply:
Hi @fgalan , @AlvaroVega ,
Could you please check the latest commit cffa723 ?